client-src/elements/chromedash-form-field.js

```diff
@@ -71,7 +74,16 @@ export class ChromedashFormField extends LitElement {
 
   firstUpdated() {
     this.initialValue = JSON.parse(JSON.stringify(this.value));
-    this.doSemanticCheck();
+
+    // We only want to do the following one time.
+    this.setupSemanticCheck();
+
+    // We need to wait until the entire page is rendered, so later dependents
+    // are available to do the semantic check, hence firstUpdated is too soon.
+    // Do first semantic check after the document is ready.
+    document.addEventListener('DOMContentLoaded', () => (setTimeout(() => {
+      this.doSemanticCheck();
+    })));
```

Comment:
I believe I've found the problem. When my page loaded, the `DOMContentLoaded` event had already fired before this event listener was added, which meant the initial render never occurred. I found [this section in the DOMContentLoaded docs](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event#checking_whether_loading_is_already_complete) that describes how to handle this situation. It worked for my problem 🙂 

```suggestion
    if (document.readyState === 'loading') {
      document.addEventListener('DOMContentLoaded', () => (setTimeout(() => {
        this.doSemanticCheck();
      })));
    } else {
      this.doSemanticCheck();
    }
```